promgen/forms.py

```diff
@@ -93,6 +93,57 @@ class Meta:
         }
 
 
+class ExporterLabelForm(ExporterForm):
+    class Meta:
+        model = models.ExporterLabel
+        exclude = ['exporter']
+
+        widgets = {
+            'name': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Name'}),
+            'value': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Value'}),
+            'is_parameter': forms.CheckboxInput()
```

Comment:
I think we **don't** want to use a parameter here (and then later build a string). I think having this as a toggle, and then try to generate strings from this later is confusing. If a user inputs a label as `foo` or `__param_foo` or `__invalid_foo` then that should be the exact name they see, and the exact name that is written into the Prometheus target.
We **do** however need to make sure we pass along the parameter when someone clicks the test version so we'll likely need to filter on that.

```python
# psudo code
params = {}
for key in labels:
  if key.startswith('__param'):
     params[ key[:7] ] = labels[key]
```

I think this will cause the least amount of confusion in the long run.